### PR TITLE
Add VS Code extension recomendations

### PR DESCRIPTION
**Pull request by jasonnutter**
When you open the repo in VS Code, it will now prompt you to install these recommendations. Feedback welcome!

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
     "eslint.quiet": true,
     "files.eol": "\n",
     "github-pr.targetBranch": "dev",
     "editor.codeActionsOnSave": {
         "source.fixAll.eslint": true
     },
+    "files.associations": {
+        "**/build/*.yml": "azure-pipelines"
+    },
+    "markiscodecoverage.searchCriteria": "**/*/coverage/lcov*.info",
```
Comment (jasonnutter) [inline .vscode/settings.json:17]:
```suggestion
    },
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,16 +1,19 @@
 {
     "editor.formatOnSave": false,
-    "eslint.autoFixOnSave": true,
```
Comment (tnorling) [inline .vscode/settings.json:3]: Any reason this was removed?

Comment (jasonnutter) [reply .vscode/settings.json:3]: Deprecated in favor of `editor.codeActionsOnSave`, which is already enabled for eslint.

**Review by tnorling — APPROVED** [review]

**Review by pkanher617 — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (jasonnutter) [inline .vscode/settings.json:17]:
```suggestion
    },
```

**Pull request merged**